node/src/chain_spec.rs

```diff
@@ -16,8 +16,6 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
The standard format is a bit different from the nightly format that we used to check in CI. You may need to either reformat your workspace (if the change is not huge) or go back to using the nightly in CI

Reply:
This is an error I inserted for rustfmt to trap ;-)